uploadNeuroDB/minc_deletion.pl

```diff
@@ -0,0 +1,142 @@
+#! /usr/bin/perl
+use strict;
+use warnings;
+use Carp;
+use Getopt::Tabular;
+use FileHandle;
+use File::Basename;
+use File::Temp qw/ tempdir /;
+use File::Path qw/ make_path /;
+use Data::Dumper;
+use FindBin;
+use Cwd qw/ abs_path /;
+
+# These are the NeuroDB modules to be used
+use lib "$FindBin::Bin";
+use NeuroDB::File;
+use NeuroDB::MRI;
+use NeuroDB::DBI;
+use NeuroDB::Notify;
+use NeuroDB::MRIProcessingUtility;
+
+my $profile   = '';      # this should never be set unless you are in a
+                             # stable production environment
+my $seriesuid = '';
+my $query     = '';
+my $selORdel  = '';
+my @opt_table = (
+                 ["Basic options","section"],
+                 ["-profile     ","string",1, \$profile,
+                  "name of config file in ../dicom-archive/.loris_mri"
+                 ],
+                 ["-seriesuid", "string", 1, \$seriesuid, "Only deletes this SeriesUID"
+                 ],
+                 );
+
+&Getopt::Tabular::GetOptions(\@opt_table, \@ARGV) || exit 1;
+
+################################################################
+################### input option error checking ################
+################################################################
+{ package Settings; do "$ENV{LORIS_CONFIG}/.loris_mri/$profile" }
+if ($profile && !@Settings::db) { 
+    print "\n\tERROR: You don't have a configuration file named ". 
+          "'$profile' in:  $ENV{LORIS_CONFIG}/.loris_mri/ \n\n"; 
+    exit 2; 
+}
+if (!$ARGV[0] || !$profile) { 
+    print " ERROR: You must type select or confirm and have an ".
+          "existing profile.\n\n";
+    exit 3;  
+}
+
+if ($ARGV[0] eq "confirm") {
+  $selORdel  = "DELETE ";
+} else {
+  $selORdel  = "SELECT * ";
+}
+
+my $data_dir         = $Settings::data_dir;
+my $dbh = &NeuroDB::DBI::connect_to_db(@Settings::db);
+
+
+sub selORdel {
```

Comment:
It's a little confusing to have a function with the same name as a variable, when they're two different things (I'm a little surprised perl allows it at all)